<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QWidget" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call A Doctor - Sign up page</string>
  </property>
  <widget class="QLineEdit" name="ClinicNameInput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name of Clinic</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>26</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 26pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Clinic Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Clinic Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Owner_NRIC">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>NRIC Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Owner NRIC </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLine1Input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Address Of Clinic 1,</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_8">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Address Line 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Address Line 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLine2Input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>590</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Address of Clinic 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendApplication">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>650</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9dff73;
color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>APPLY</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>185</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Clinic contact number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContactNumberInput">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Contact Number Of Clinic</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Clinic Email  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClinicEmailInput">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Clinic Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Username&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Username</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome_label_16">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VerifyEmail">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9dff73;
color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="SpecialtyList">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>General Practice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cardiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cardiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dermatology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Endocrinology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neurology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Obstetrics and Gynecology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orthopedic Surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pediatrics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Psychiatry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Surgery (General)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Urology</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Welcome_label_7">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>230</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;First name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Clinic Specialty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
